### PR TITLE
refactored to new logging

### PR DESCRIPTION
**Pull request by OttoVT**

### CODE REVIEW

File: src/Lykke.Job.BlockchainCashinDetector/Workflow/CommandHandlers/EnrollToMatchingEngineCommandsHandler.cs [hunk 1]
```diff
@@ -54,7 +54,7 @@ public async Task<CommandHandlingResult> Handle(EnrollToMatchingEngineCommand co
             // First level deduplication just to reduce traffic to the ME
             if (await _deduplicationRepository.IsExistsAsync(command.OperationId))
             {
-                _log.WriteInfo(nameof(EnrollToMatchingEngineCommand), command.OperationId, "Deduplicated at first level");
+                _log.Info("Deduplicated at first level");
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.BlockchainCashinDetector/Workflow/CommandHandlers/EnrollToMatchingEngineCommandsHandler.cs]: pass command as context

File: src/Lykke.Job.BlockchainCashinDetector/Workflow/CommandHandlers/EnrollToMatchingEngineCommandsHandler.cs [hunk 2]
```diff
@@ -88,7 +88,7 @@ public async Task<CommandHandlingResult> Handle(EnrollToMatchingEngineCommand co
                 case MeStatusCodes.Duplicate:
                     if (cashInResult.Status == MeStatusCodes.Duplicate)
                     {
-                        _log.WriteInfo(nameof(EnrollToMatchingEngineCommand), command.OperationId, "Deduplicated by the ME");
+                        _log.Info("Deduplicated by the ME");
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.BlockchainCashinDetector/Workflow/CommandHandlers/EnrollToMatchingEngineCommandsHandler.cs]: pass command as context

File: src/Lykke.Job.BlockchainCashinDetector/Workflow/CommandHandlers/EnrollToMatchingEngineCommandsHandler.cs [hunk 3]
```diff
@@ -111,7 +111,7 @@ public async Task<CommandHandlingResult> Handle(EnrollToMatchingEngineCommand co
 
                 default:
                     // Just abort cashin for futher manual processing. ME call could not be retried anyway if responce was received.
-                    _log.WriteWarning(nameof(EnrollToMatchingEngineCommand), command.OperationId, $"Unexpected response from ME. Status: {cashInResult.Status}, ME message: {cashInResult.Message}");
+                    _log.Warning($"Unexpected response from ME. Status: {cashInResult.Status}, ME message: {cashInResult.Message}", null, command.OperationId);
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.BlockchainCashinDetector/Workflow/CommandHandlers/EnrollToMatchingEngineCommandsHandler.cs]: pass entire command all context, not only OperationId

File: src/Lykke.Job.BlockchainCashinDetector/Workflow/CommandHandlers/EnrollToMatchingEngineCommandsHandler.cs (hunk 1 above)
Comment (OttoVT) [reply src/Lykke.Job.BlockchainCashinDetector/Workflow/CommandHandlers/EnrollToMatchingEngineCommandsHandler.cs]: fixed

**Pull request merged**